systemtest/src/test/java/io/strimzi/systemtest/upgrade/StrimziDowngradeST.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.upgrade;
+
+import io.strimzi.systemtest.utils.StUtils;
+import io.vertx.core.json.JsonObject;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.io.IOException;
+
+import static io.strimzi.systemtest.Constants.INTERNAL_CLIENTS_USED;
+import static io.strimzi.systemtest.Constants.UPGRADE;
+import static org.junit.jupiter.api.Assumptions.assumeTrue;
+
+@Tag(UPGRADE)
+public class StrimziDowngradeST extends AbstractUpgradeST {
```

Comment:
Maybe a javadoc to clarify that this is about upgrading Strimzi, and the Kafka upgrade/downgrade is done in `KafkaUpgradeDowngradeST`.